back/scripts/datasets/marches.py

```diff
@@ -147,13 +157,24 @@ def unnest_marche(declaration: dict):
         if isinstance(titulaires, dict):
             titulaires = [titulaires]
 
+        # titulaire is sometines nested
+        if titulaires[0] is not None and "titulaire" in titulaires[0].keys():
+            titulaires = [titu["titulaire"] for titu in titulaires]
+
+        # acheteur is sometimes just a dictionnary with a single "id" key
         unnested = {}
         for k, v in local_decla.items():
-            if isinstance(v, (list, dict)):
-                v = json.dumps(v)
-            elif isinstance(v, decimal.Decimal):
-                v = float(v)
-            unnested[k] = v
+            if k == "acheteur":
+                try:
+                    unnested[k + ".id"] = str(v["id"])
```

Comment:
at this point we know `k == "acheteur"` 
```suggestion
                    unnested["acheteur.id"] = str(v["id"])
```

Reply:
Done

Reply:
still there :)

Reply:
My bad, it should no longer be there

Reply:
My bad, it should no longer be